sihl/src/web_trailing_slash.ml

```diff
@@ -1,10 +1,14 @@
 let middleware () =
   let filter handler req =
-    let uri = req.Opium.Request.target |> Uri.of_string in
+    let target = req.Opium.Request.target in
```

Comment:
You could keep this as one let binding, I'd say

Reply:
Agree, fixed.

Reply:
Thx!